README.md

```diff
@@ -182,3 +182,5 @@ Examples
 
     $aliases = $dir->users_aliases->listUsersAliases($users_email);
     $alias = $dir->users_aliases->delete($users_email, $alias);
+
+### DEVELOPER'S NOTE: Releases should include updating the version in composer.json to match
```

Comment:
I know Composer/Packagist can use git tags to find the version. Does something need the version to be in the composer.json file?

Reply:
The build. Otherwise it gives an annoying message.